src/docs/Contributing/Principles/ContributionJourney/index.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: 'Contribution journey'
+order: 1
+---
+
+Mozaic is a design system that wants to make designers and developers collaboration easy. When you contribute to Mozaic, you make the work you need, you don't wait for our team to make this work available for you.
```

Comment:
**When you contribute to Mozaic, you make the work you need, you don't wait for our team to make this work available for you.**

I don't find this sentence very meaningful, perhaps it can be rephrased into something like this: 

_By contributing to Mozaic, you become an actor of this project and can create the elements you need, without necessarily waiting for them to be made available to you by the Mozaic team._